src/client/components/Reservations.js

```diff
@@ -0,0 +1,124 @@
+import React, { useEffect, useState } from "react";
+import "./meals.css";
+
+function Reservations() {
+  const [reservationState, setReservationState] = useState([]);
+  const [success, setSuccess] = useState("");
+  const [message, setMessage] = useState("");
+
+  const [foodId, setFoodId] = useState("");
+  const [name, setName] = useState("");
+  const [phone, setPhone] = useState("");
+  const [email, setEmail] = useState("");
+  const [guest, setGuest] = useState("");
+  useEffect(() => {
+    fetchReservation();
+  }, []);
+  const fetchReservation = async () => {
+    const data = await fetch("/api/meals?availableReservations");
+    const items = await data.json();
+    setReservationState(items);
+  };
+  console.log(reservationState);
+  const mappedReservation = reservationState.map((reservation) => (
+    <>
+      <section className="menu-item">
+        <p>Id: {reservation.id}</p>
+        <p>Name: {reservation.title}</p>
+        <p>Booked Seats: {reservation.sum}</p>
+        <p>Description: {reservation.description}</p>
+        <p>Max Seats: {reservation.max_reservations}</p>
+      </section>
+    </>
+  ));
+  function addShift(e) {
+    e.preventDefault();
+    const data = {
+      meal_id: Number(foodId),
+      contact_name: name,
+      contact_phonenumber: Number(phone),
+      contact_email: email,
+      number_of_guests: Number(guest),
+      created_date: "2022-04-03",
```

Comment:
I think this date should be current time when user make a reservation. So you can search on google to get current time easily. For example `new Date().toISOString().split('T')[0]`